data_summaries/WildChat.json

```diff
@@ -0,0 +1,94 @@
+{
+  "wc-gpt-3.5-turbo": {
```

Comment:
Let's call it "wildchat-gpt_turbo" ("-" separates the collection from dataset name, "_" separates words in either name)